src/libclient/CMakeLists.txt

```diff
@@ -0,0 +1,175 @@
+project(libowncloudclient)
```

Comment:
what's wront with the `libowncloudclient` name?  It is an internal name only so branding should not care.  Or am i missing something.